packages/components/modules/activity-log/ActivityLogComponent/index.tsx

```diff
@@ -0,0 +1,62 @@
+'use client'
+
+import { ChangeEventHandler, FC, useState, useTransition } from 'react'
+
+import { Searchbar } from '@baseapp-frontend/design-system'
+
+import { Box, Typography } from '@mui/material'
+import { useForm } from 'react-hook-form'
+
+import SearchNotFoundState from '../../messages/SearchNotFoundState'
+import { useActivityLogs } from '../graphql/queries/ActivityLogsFragment'
+import EventFilterChip from './EventFilterChip'
+import DefaultLogGroups from './LogGroups'
+import { ActivityLogComponentProps } from './types'
+
+const ActivityLogComponent: FC<ActivityLogComponentProps> = ({
+  queryRef,
+  LogGroups = DefaultLogGroups,
+  LogGroupsProps,
+}) => {
+  const [selectedFilters, setSelectedFilters] = useState<string[]>(['All'])
+  const [isPending] = useTransition()
+  const { control, watch } = useForm({ defaultValues: { search: '' } })
+  const searchValue = watch('search')
+  const { logGroups, loadNext, hasNext, isLoadingNext } = useActivityLogs(queryRef)
+  const handleSearchChange: ChangeEventHandler<HTMLInputElement> = () => {}
```

Comment:
maybe add a `TODO: use startTransition from useTransition when triggering a refetch` so we don't forget to use it